README.md

```diff
@@ -21,6 +21,13 @@ Config.php parameters to operate the server in master mode:
 
 // operation mode
 'gss.mode' => 'master',
+
+// define a master admin, this user will not be redirected to a slave but is
+// allowed to login at the master to perform administration tasks
+'gss.master.admin' => 'adminUid',
```

Comment:
shouldn't it just not redirect on any user that is available locally? there might be more admins after all.

Reply:
hm, not sure. In case of external backends you might not know the users before they login the first time. So you have to decide on first login what to do. Therefore I might want to make it more explicit. But I'm fine with changing it from a string to an array.

Reply:
the portal is not suppose to serve any regular users, or is it?